bin/dhttpd.dart

```diff
@@ -1,3 +1,40 @@
-import '_boot.dart';
+import 'dart:io';
```

Comment:
[nit] blank line between `dart:` and `package:` imports

Reply:
ack